rollup-interface/specs/interfaces/stf.md

```diff
@@ -1,270 +1,60 @@
 # State transition Function
 
 ## Overview
-
-The state transition function is the "business logic" of a rollup.
-It is defined in a particular zkVM (since implementation details like hash and proof formats functions need to change). However,
-an implementation of an STF may be reused across multiple DA layers.
+The state transition function serves as the "business logic" for a rollup. It specifies the methods that will be automatically invoked by the rollup during different stages of block processing. Implementations of this trait can be integrated with any ZKVM and DA Layer resulting in a fully functional rollup.
 
 The Sovereign SDK guarantees that all relevant transactions will be delivered to the STF for processing
-exactly once, in the order that they appear on the DA layer. The STF is responsible for implementing its own metering
-and billing (to prevent spam), and for maintaining a "consensus set" (a list of addresses who are allowed to post transactions).
+exactly once, in the order that they appear on the DA layer. The STF is responsible for implementing its own metering and billing (to prevent spam), and for maintaining a "consensus set" (a list of addresses who are allowed to post transactions).
```

Comment:
I know this is not specific to changes in this PR, but I wonder if we can find another name for "consensus set". It might make people wonder that if we introduce an additional consensus. How's "sequencer set"?